pymc/model/core.py

```diff
@@ -1241,6 +1241,15 @@ def register_rv(
             self.add_named_variable(rv_var, dims)
             self.set_initval(rv_var, initval)
         else:
+            if (
```

Comment:
this should have checked if minibatch is anywhere in the ancestors of value not just immediately

Reply:
Do we need to do that? This check is only for observed random variables. I thought for those we don't allow the value to depend on another node